recipe/meta.yaml

```diff
@@ -19,14 +19,16 @@ requirements:
         - numpy x.x
         - cython
         - flann
-        - gcc 4.8*  # [osx]
+        - isl 0.12.*   # [osx]
+        - gcc 4.8*     # [osx]
     run:
         - python
         - numpy x.x
         - scipy >=0.16
         - scikit-learn >=0.17
         - pyamg
         - pyflann
+        - isl 0.12.*   # [osx]
         - libgcc 4.8*  # [osx]
```

Comment:
If you unpin `gcc` you need to unpin `libgcc` too :smile_cat:  (Sorry it seems almost like I am putting you on a fools errands on purpose, but I assure you I am just a bad reviewer :stuck_out_tongue_winking_eye::)